chain/contractWrappers.go

```diff
@@ -233,9 +233,36 @@ func (ph *VotingHandle) SetStatusTxArgs(ctx context.Context, pid [types.ProcessI
 	return setprocessTxArgs, nil
 }
 
-// SetCensusTxArgs
-func (ph *VotingHandle) SetCensusTxArgs(ctx context.Context, pid [types.ProcessIDsize]byte) (*models.SetProcessTx, error) {
-	return nil, fmt.Errorf("not implemented")
+// SetCensusTxArgs returns a SetProcess tx instance with census mkroot and census mkURI set
+func (ph *VotingHandle) SetCensusTxArgs(ctx context.Context, pid [types.ProcessIDsize]byte, namespace uint16) (*models.SetProcessTx, error) {
+	processData, err := ph.VotingProcess.Get(&ethbind.CallOpts{Context: ctx}, pid)
+	if err != nil {
+		return nil, fmt.Errorf("error fetching process from Ethereum: %w", err)
+	}
+	// create setProcessTx
+	setprocessTxArgs := new(models.SetProcessTx)
+	// process id
+	setprocessTxArgs.ProcessId = pid[:]
+	// process mkRoot
+	if setprocessTxArgs.CensusRoot, err = hex.DecodeString(util.TrimHex(processData.MetadataCensusRootCensusUri[1])); err != nil {
+		return nil, fmt.Errorf("invalid mkRoot: %w", err)
```

Comment:
Its not a "mkRoot" anymore but a "censusRoot"

Reply:
changed on all files :)